coderd/database/migrations/000127_ha_coordinator.up.sql

```diff
@@ -0,0 +1,95 @@
+BEGIN;
+
+CREATE TABLE tailnet_coordinators (
+	id uuid NOT NULL PRIMARY KEY,
+	heartbeat_at timestamp with time zone NOT NULL
+);
```

Comment:
Could this be integrated into the replicas table somehow? Feels unnecessary to have both essentially doing the same thing (heartbeating).